### ISSUE TITLE
Model not reloading, no matter what I try

### ISSUE THREAD

**Issue by Iciir** [bonastreyair/node-red-contrib-teachable-machine#32]
From the same picture I am getting result "100% A" in TeachableMachine, but in node red I am getting 100% B.  (which I was getting before training it more).

It seems to completely ignore any changes I've done to the training.

Is there a file of the model downloaded somewhere that I could delete so that it re-downloads the model?

Disabling does not fix it, deleting the node and reinserting another does not fix it, giving it an invalid URL and then the correct one does not fix it.

EDIT:  The problem is definitely there because I changed the name of class A, to something else, and it's still using the old name.

**Comment by bonastreyair** [comment]
For how you describe it, it seems the label does not get updated if you try to reload it... 

Exactly, if you send a message with a `msg.reload` field it supposedly to re-download and reload all the files. There should be an example template.

- Could you check that using a different model with different labels those also change if you change the URL of a different model? 
- Also check the URL is the right one.

I've tested this on my side and I don't have any issues, reloading the model with different labels works as expected. Model in teachable machine behaves equally as using the node-red node. I would need more information to reproduce your bug, or find out what is being bad configured.

Thanks!

**Comment by Iciir** [comment]
Yes it seems to reload successfully, just that new labels are a bit delayed in updating.  I just updated the model and some labels, and reloaded, let me check if the new label appears with a test right now.

[After the test]  The new label is updated, **everything works perfectly fine**.  I don't know why it was delayed yesterday, maybe because of the ways I messed with it before discovering `msg.reload`.

By the way, my set-up for reloading, for maybe future readers, is like this:

![image](https://user-images.githubusercontent.com/93911948/141299507-71f70c57-71a5-4a5d-93e7-027b4f01f1d7.png)

![image](https://user-images.githubusercontent.com/93911948/141299552-6e429d4a-5b4c-4956-a273-3a6907259499.png)

Simple like that.  :)

Thank you for responding @bonastreyair
